Description: Remove the mimetype registration in-game
 Because the package does mimetype registration already,
 there should be no way to redo the process in-game.
Forwarded: not-needed
Author: Aditya Vaidya <kroq.gar78@gmail.com>
Last-Update: 2015-04-11
--- b/src/gui/game/GameController.cpp
+++ a/src/gui/game/GameController.cpp
@@ -731,11 +731,11 @@ void GameController::Tick()
 		((LuaScriptInterface*)commandInterface)->Init();
 #endif
 #if !defined(MACOSX) && !defined(NO_INSTALL_CHECK)
-		if(!Client::Ref().GetPrefBool("InstallCheck", false))
+		/*if(!Client::Ref().GetPrefBool("InstallCheck", false))
 		{
 			Client::Ref().SetPref("InstallCheck", true);
 			Install();
-		}
+		}*/
 #endif
 		firstTick = false;
 	}
